pensieve/statistics.py

```diff
@@ -167,3 +168,70 @@ def transform(self, df: DataFrame, metric: str) -> "StatisticResultCollection":
             )
 
         return stats_results
+
+
+@attr.s(auto_attribs=True)
+class Binomial(Statistic):
+    ref_branch_label: str = "control"
+    confidence_interval: float = 0.95
+
+    def transform(self, df: DataFrame, metric: str) -> "StatisticResultCollection":
+        stats_results = StatisticResultCollection([])
+
+        critical_point = (1 - self.confidence_interval) / 2
+        summary_quantiles = (critical_point, 1 - critical_point)
+
+        ma_result = mozanalysis.bayesian_stats.binary.compare_branches(
+            df,
+            col_label=metric,
+            ref_branch_label=self.ref_branch_label,
+            individual_summary_quantiles=summary_quantiles,
+            comparative_summary_quantiles=summary_quantiles,
+        )
+
+        for branch, branch_result in ma_result["individual"].items():
+            lower, upper = _extract_ci(branch_result, critical_point)
+            result = StatisticResult(
+                metric=metric,
+                statistic="binomial",
+                parameter=None,
+                branch=branch,
+                ci_width=self.confidence_interval,
+                point=branch_result["mean"],
+                lower=lower,
+                upper=upper,
+            )
+            stats_results.data.append(result)
+
+        for branch, branch_result in ma_result["comparative"].items():
+            lower_abs, upper_abs = _extract_ci(branch_result["abs_uplift"], critical_point)
+            stats_results.data.append(
+                StatisticResult(
+                    metric=metric,
+                    statistic="binomial",
+                    parameter=None,
+                    branch=branch,
+                    comparison_to_control="difference",
+                    ci_width=self.confidence_interval,
+                    point=branch_result["abs_uplift"]["exp"],
+                    lower=lower_abs,
+                    upper=upper_abs,
+                )
+            )
+
+            lower_rel, upper_rel = _extract_ci(branch_result["rel_uplift"], critical_point)
+            stats_results.data.append(
+                StatisticResult(
+                    metric=metric,
+                    statistic="binomial",
+                    parameter=None,
+                    branch=branch,
+                    comparison_to_control="relative_uplift",
```

Comment:
Just wondering, will these comparisons end up being a relatively fixed set? If so, we could have some kind of `enum` for all the types of comparisons. That would make it easier to keep track of consistent naming.

Reply:
Hm, probably! This will be important to resolve once we decide how stats should interact with the dashboarding component. I think I'd like to punt for now.